cristin-project/src/main/java/no/unit/nva/cristin/projects/model/nva/HealthProjectData.java

```diff
@@ -15,32 +15,32 @@ public class HealthProjectData implements JsonSerializable {
     public static final String CLINICAL_TRIAL_PHASE = "clinicalTrialPhase";
 
     @JsonProperty(TYPE)
-    private final String type;
+    private final HealthProjectType type;
     @JsonProperty(LABEL)
     private final Map<String, String> label;
     @JsonProperty(CLINICAL_TRIAL_PHASE)
-    private final String clinicalTrialPhase;
+    private final ClinicalTrialPhase clinicalTrialPhase;
 
     /**
      * Nva model for a project having additional health related data.
      */
     @JsonCreator
-    public HealthProjectData(@JsonProperty(TYPE) String type, @JsonProperty(LABEL) Map<String, String> label,
-                             @JsonProperty(CLINICAL_TRIAL_PHASE) String clinicalTrialPhase) {
+    public HealthProjectData(@JsonProperty(TYPE) HealthProjectType type, @JsonProperty(LABEL) Map<String, String> label,
```

Comment:
@brinxmat Is this correct. Should i remove JsonProperty from constructors?
@StigNorland I sometimes need to generate new values when writing unit tests, using the constructor. I think i will keep a parameterized constructor for now since I do not have any setters.

Reply:
Immutable classes are a good thing in many ways. Why allow changing class instance fields with setters if they are just DTO/model classes? It only allows for doing bad things if you ask me. This is the reason I mostly user final on instance fields and set them all in the constructor. A builder can help if there too many fields.